### PR TITLE
Language panel for contacts

### PR DESCRIPTION
**Pull request by jul4ajka**
Language panel for contact page

### CODE REVIEW

File: src/components/forms/contacts-form/contacts-form.js [hunk 1]
```diff
@@ -82,12 +86,15 @@ const ContactsForm = ({ contactSaveHandler, initialValues }) => {
       .required(ENTER_PHONE_NUMBER_MESSAGE),
     uaSchedule: Yup.string()
       .min(10, INPUT_LENGTH_MESSAGE)
+      .matches(uaRegex, ENTER_UA_SCHEDULE_MESSAGE)
       .required(ENTER_UA_SCHEDULE_MESSAGE),
     enSchedule: Yup.string()
       .min(10, INPUT_LENGTH_MESSAGE)
+      // .matches(enRegex, ENTER_EN_SCHEDULE_MESSAGE)
```
Comment (AndriyLymych) [inline src/components/forms/contacts-form/contacts-form.js:93]: remove commented line

File: src/components/forms/contacts-form/contacts-form.js [hunk 2]
```diff
@@ -121,6 +128,19 @@ const ContactsForm = ({ contactSaveHandler, initialValues }) => {
...
+  ];
+
+  const inputOptions = {
+    errors,
+    touched,
+    handleChange,
+    values,
+    inputs
+  };
+  console.log(errors);
```
Comment (AndriyLymych) [inline src/components/forms/contacts-form/contacts-form.js:143]: remove log

File: src/components/forms/contacts-form/contacts-form.js [hunk 3]
```diff
@@ -248,8 +293,8 @@ const ContactsForm = ({ contactSaveHandler, initialValues }) => {
                   helperText={touched.enSchedule && errors.enSchedule}
                 />
               </Paper>
```
Comment (AndriyLymych) [inline src/components/forms/contacts-form/contacts-form.js:295]: remove commented code

File: src/components/forms/contacts-form/contacts-form.js [hunk 4]
```diff
@@ -285,24 +330,8 @@ const ContactsForm = ({ contactSaveHandler, initialValues }) => {
                   error={touched.enAddress && !!errors.enAddress}
                   helperText={touched.enAddress && errors.enAddress}
                 />
```
Comment (AndriyLymych) [inline src/components/forms/contacts-form/contacts-form.js:332]: same

**Review by AndriyLymych — APPROVED** [review]

**Review by kapoliub — APPROVED** [review]

**Review by moran711 — APPROVED** [review]

**Pull request merged**